lib/log.rb

```diff
@@ -183,6 +183,21 @@ def reset_dedup
       @dup_buffer.reset if @dup_buffer
     end
 
+    private
+    def sensor(msg)
+      secured_lines = []
+      lines = msg.split("\n")
+      sensor_kw = %W{client_id  client_secret  subscription_id  tenant_id  access_key_id  secret_access_key secret authorization password oauth token}
```

Comment:
.dockercfg
.dockerconfigjson
kubeConfig
htpasswd
ca.crt / service-ca.crt
tls.crt
tls.key
service-account.key

Reply:
Thank you Liang!
Is there an example of one ore more of the above format available? In the code, I was assuming the texts were in this form`client_id: xxxx`, just wanted to make sure we can cover more forms and keywords.

Reply:
Yes, all of them are in format `foo: bar`. E.g,
```
- apiVersion: v1
  data:
    kubeConfig: YXBpVmVyc2lvbjogdjE***
- apiVersion: v1
  data:
    .dockerconfigjson: eyJhdXRocyI6eyJicmV3LnJlZ2lzdHJ***
```